src/UI/Helpers/IPv4Helper.h

```diff
@@ -0,0 +1,25 @@
+/*
+	Copyright 2021 OpenJAUS, LLC (dba MechaSpin). Subject to the MIT license.
+*/
+
+#pragma once
+
+#include <QChar>
+#include <QPlainTextEdit>
+
+#include <vector>
+
+class IPv4Helper
```

Comment:
With the move of the address to byte array function out of this class, it feels like IPv4Validator or IPv4Validation would be a better name here since all the functions relate to validation.